### ISSUE TITLE
How to "Build" the Docker Container ?

### ISSUE THREAD

**Issue by ElryWeeb** [C9Glax/tranga-website#56]
Im trying to use @db-2001 [settings repo](https://github.com/C9Glax/tranga-website/pull/41), but im failing to understand on how to deploy this correctly from source.
Im using Docker and my compose file looks like this:

```version: '3'
services:
  tranga-api:
    image: glax/tranga-api:cuttingedge
    container_name: tranga-api
    volumes:
      - /root/manga:/Manga
      - ./settings:/usr/share/tranga-api
    ports:
      - "127.0.0.1:6531:6531"
    restart: unless-stopped
  tranga-website:
    #image: glax/tranga-website:cuttingedge
    build: ./tranga-website
    container_name: tranga-website
    ports:
      - "127.0.0.1:3000:80"
    depends_on:
      - tranga-api
    restart: unless-stopped
```
i cloned db's settings repo into ./tranga-website and it does seem to look like docker is building it and starting it,
yet it doesnt seem to work and i get a blank nginx page when accessing the tranga website url.

**Comment by db-2001** [comment]
There is a reason my branch hasn't been merged into cutting edge or master yet. I'm still waiting on API changes on @C9Glax side. The website is currently incompatible with the existing APi. 

If you insist on going this route, you won't have full functionality, but make sure you are cloning the repo into the same place where the tranga-website container stores the original files AND DELETE the original files. The webserver doesn't know what to do with 2 index.htmls

Finally as a last point, you don't need 127.0.0.1 in your docker compose files. "6531:6531" will work just fine for exposing the port.

 You also said you were accessing the website via a URL, this may imply you have a reverse proxy set up. Make sure the reverse-proxy URL is pointed to the correct IP:PORT combination, which in your case should be <HOST_IP>:3000

**Comment by db-2001** [comment]
@ElryWeeb it has now been merged in, feel free to use the official docker image to try it out :)

**Comment by ElryWeeb** [comment]
Thank you for the info
